src/components/MyReservation.astro

```diff
@@ -0,0 +1,48 @@
+---
+
+---
+
+<div class="w-full">
+  <h2 class="mb-5 text-4 font-bold text-center text-cph-blue">
+    Min reservation
+  </h2>
+  <form>
+    <div class="flex gap-4 justify-center px-10">
+      <div class="mb-6 w-full">
+        <input
+          type="email"
+          id="email"
+          class="block w-full p-3 border border-grey-lighter rounded-md h-14"
+          placeholder="Din e-mailadresse"
+        />
+        <p class="text-small">
+          Den e-mail du brugte, da du foretog din reservation
+        </p>
+      </div>
+      <div class="mb-6 w-full">
+        <input
+          type="text"
+          id="reservationNumber"
+          class="block w-full p-3 border border-grey-lighter rounded-md h-14"
+          placeholder="Dit reservationsnummer"
+        />
+        <p class="text-small">
+          Du finder reservationsnummer i din ordrebekræftelse
+        </p>
+      </div>
+    </div>
+
+    <div class="flex justify-center px-10 flex-col">
+      <button
```

Comment:
den bliver lidt lang pt.
![image](https://github.com/Ballademager/cph/assets/118718923/f93ddbf7-65be-46a7-b34e-e55f2ae8a9ea)

Reply:
`width: fit-content` gør det for mig
og så skal den måske have lidt mere inline-padding efter ovenstående bliver gjort